pkg/tls/tlsmanager.go

```diff
@@ -217,6 +217,12 @@ func buildTLSConfig(tlsOption Options) (*tls.Config, error) {
 		conf.MinVersion = minConst
 	}
 
+	// Set the maximum TLS version if set in the config TOML
+	if maxConst, exists := MaxVersion[tlsOption.MaxVersion]; exists {
+		conf.PreferServerCipherSuites = true
```

Comment:
Could you explain this line?

Reply:
I was just trying to replicate the way the minVersion option is handled.  

I used same code as block above that checks if tlsOptions.MinVersion is set in config and adds to the conf structure. 

That code set the PreferServerCipherSuites to true so I kept that as well.

That should probably be separately configurable I would think.

The PreferServerCipherSuites setting can be removed and still achieve my objective.

Reply:
thank you, you can keep this line.